pages/stands/index.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { graphql } from 'react-relay';
+import { withDataAndLayout, WithDataAndLayoutProps } from '../../lib/withData';
+import PageView from '../../components/PageView';
+import styled from 'styled-components';
+
+import { stands_QueryResponse } from '../../__generated__/stands_Query.graphql';
+import { useEffect, useState } from 'react';
+import StandCard from '../../components/Stands/StandCard';
+import { currentDayCompanies, timeIsAfterNow } from '../../utils/time';
+import LivePlayer from '../../components/Stands/LivePlayer';
+
+// Update the currentEvent-list every 30 sec
+const intervalLength = 1000 * 30;
+
+const Index = ({
+  error,
+  props,
+}: WithDataAndLayoutProps<stands_QueryResponse>): JSX.Element => {
+  const [time, setTime] = useState(Date.now());
+  const {
+    mainCollaborator,
+    collaborators,
+    startDate,
+    endDate,
+  } = props.currentMetaData;
+
+  useEffect(() => {
+    const interval = setInterval(() => setTime(Date.now()), intervalLength);
+    return (): void => clearInterval(interval);
+  }, [props.currentMetaData]);
+
+  const currentDayCompaniesIds = (): string[] => {
+    return props.currentMetaData[currentDayCompanies(endDate)]
+      ? props.currentMetaData[currentDayCompanies(endDate)]!.map((c) => c.id)
```

Comment:
Pls no non-null assertion, this should be handled somewhere.

Reply:
And the string key accessing here is not really a good option here and goes around types.
A suggestion is to instead do something like 
```jsx
let currentDayCompanies = [];
if (isFirstDay(currentMetadata...)) {
currentDayCompanies = props.currentMetadata.companiesFirstDay;
}else if (isLastDay(currentMetadata)) {
 currentDayCompanies = props.currentMetadata.companiesLastDay;
};
```

Is something with terneries, but it won't be as readable I think. This avoids hardkoding the object keys in the `currentDayCompanies` as well as should fix the null checks.